Tests/modules/network_related/_ssl_test.py

```diff
@@ -1,385 +0,0 @@
-             0x10000000)
-
-
-def test_OP_NO_TLSv1_2():
-    AreEqual(real_ssl.OP_NO_TLSv1_2,
-             0x8000000)
-
-def test_RAND_add():
-    #--Positive
-    AreEqual(real_ssl.RAND_add("", 3.14),
-             None)
-    AreEqual(real_ssl.RAND_add(u"", 3.14),
-             None)
-    AreEqual(real_ssl.RAND_add("", 3),
-             None)
-    
-    #--Negative
-    for g1, g2 in [ (None, None),
-                    ("", None),
-                    (None, 3.14), #http://ironpython.codeplex.com/WorkItem/View.aspx?WorkItemId=24276
-                    ]:
-        AssertError(TypeError, real_ssl.RAND_add, g1, g2)
-    
-    AssertError(TypeError, real_ssl.RAND_add)
-    AssertError(TypeError, real_ssl.RAND_add, "")
-    AssertError(TypeError, real_ssl.RAND_add, 3.14)
-    AssertError(TypeError, real_ssl.RAND_add, "", 3.14, "")
-
-
-def test_RAND_status():
-    #--Positive
-    AreEqual(real_ssl.RAND_status(),
-             1)
-             
-    #--Negative
-    AssertError(TypeError, real_ssl.RAND_status, None)
-    AssertError(TypeError, real_ssl.RAND_status, "")
-    AssertError(TypeError, real_ssl.RAND_status, 1)
-    AssertError(TypeError, real_ssl.RAND_status, None, None)
-
-    
-def test_SSLError():
-    AreEqual(real_ssl.SSLError.__bases__, (socket.error, ))
-
-
-def test_SSL_ERROR_EOF():
-    AreEqual(real_ssl.SSL_ERROR_EOF, 8)
-
-
-def test_SSL_ERROR_INVALID_ERROR_CODE():
-    AreEqual(real_ssl.SSL_ERROR_INVALID_ERROR_CODE, 9)
-
-
-def test_SSL_ERROR_SSL():
-    AreEqual(real_ssl.SSL_ERROR_SSL, 1)
-
-
-def test_SSL_ERROR_SYSCALL():
-    AreEqual(real_ssl.SSL_ERROR_SYSCALL, 5)
-
-
-def test_SSL_ERROR_WANT_CONNECT():
-    AreEqual(real_ssl.SSL_ERROR_WANT_CONNECT, 7)
-
-
-def test_SSL_ERROR_WANT_READ():
-    AreEqual(real_ssl.SSL_ERROR_WANT_READ, 2)
-
-
-def test_SSL_ERROR_WANT_WRITE():
-    AreEqual(real_ssl.SSL_ERROR_WANT_WRITE, 3)
-
-
-def test_SSL_ERROR_WANT_X509_LOOKUP():
-    AreEqual(real_ssl.SSL_ERROR_WANT_X509_LOOKUP, 4)
-
-
-def test_SSL_ERROR_ZERO_RETURN():
-    AreEqual(real_ssl.SSL_ERROR_ZERO_RETURN, 6)
-
-
- #http://ironpython.codeplex.com/WorkItem/View.aspx?WorkItemId=21411
-def test___doc__():
-    expected_doc = """Implementation module for SSL socket operations.  See the socket module
-for documentation."""
-    AreEqual(real_ssl.__doc__, expected_doc)
-
-
-def test__test_decode_cert():
-    if not is_cpython and hasattr(real_ssl, "decode_cert"):
-        raise Exception("Please add a test for _ssl.decode_cert")
-    print 'TODO: no implementation to test yet.'
-
-
-def test_sslwrap():
-    print 'TODO: no implementation to test yet.'
-
-
-def test_SSLType():
-    #--Positive
-    if is_cpython:
-        AreEqual(str(real_ssl.SSLType),
-                 "<type 'ssl.SSLContext'>")
-    else:
-        #http://ironpython.codeplex.com/WorkItem/View.aspx?WorkItemId=24266
-        AreEqual(str(real_ssl.SSLType),
-                 "<type 'socket.ssl'>")
-                 
-    #--Negative
-    if is_cpython:
```

Comment:
Ah, that's because there is no SSLType for the _ssl module anymore. Both IronPython and CPython get a message saying the module object has no attribute SSLType, so the test was bogus.